.github/workflows/integration-tests-github-tier-0.yml

```diff
@@ -0,0 +1,73 @@
+name: GitHub integration tests tier 0
+
+on:
+  pull_request:
```

Comment:
Add possibility to launch it manually:
https://github.com/wazuh/wazuh/blob/master/.github/workflows/integration-tests-analysisd-tier-0-1.yml#L4